starknet_py/net/schemas/rpc.py

```diff
@@ -479,12 +512,52 @@ class StarknetBlockWithTxHashesSchema(Schema):
     l1_gas_price = fields.Nested(
         ResourcePriceSchema(), data_key="l1_gas_price", required=True
     )
+    l1_data_gas_price = fields.Nested(
+        ResourcePriceSchema(), data_key="l1_data_gas_price", load_default=None
+    )
+    l1_da_mode = DaModeTypeField(data_key="l1_da_mode", load_default=None)
 
     @post_load
     def make_dataclass(self, data, **kwargs) -> StarknetBlockWithTxHashes:
         return StarknetBlockWithTxHashes(**data)
 
 
+class TransactionWithReceiptSchema(Schema):
+    transaction = fields.Nested(TransactionSchema, data_key="transaction")
+    receipt = fields.Nested(TransactionReceiptSchema, data_key="receipt")
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> TransactionWithReceipt:
+        return TransactionWithReceipt(**data)
+
+
+class StarknetBlockWithReceiptsSchema(Schema):
```

Comment:
Could we create a parent schema that could also be reused for `StarknetBlockWithTxHashesSchema`?